docs/review-merge-process.md

```diff
@@ -0,0 +1,23 @@
+# Mappings Review / Merge Process
+
+This document details the review and merge process for Parchment mappings.
+
+### Requirements to merge:
+
+- If we have any automatic tests, they all must pass.
+  - Example: A PR may not be merged if the Contributor License Agreement (CLA) is not signed.
+- A PR requires two approving reviews
+  - One approving review is required to be from a Parchment team member.
+- We ask that people don’t review things that they don’t actually have the capability to effectively review (knowledge wise).
+- All external reviewers are welcome, but not required.
+
+On Resolving conflicting opinions:
+
+- Any names involving substantial conflicts should be available for public input for some length of time (i.e. a discussion time) before merging. After such time, the PR may be merged if there is:
+  1. A simple majority of Parchment team reviewers
+  2. In the event of a tie, A simple majority of all reviewers
```

Comment:
What does "all reviewers" mean in this context? Because the first point specifies that a simple majority of all Parchment team reviewers, then it follows that the second point must be wider in scope than "all Parchment team reviewers". 
 - Does this mean that external (non mapping-team) reviewers count? 
 - Does this mean that all mapping team members (not just those in the reviewers subteam) are counted? 
 - What happens if we reasonably can't get the votes of a majority of the reviewers due to lack of activity on their part?

We should outline this process a bit more in a separate document (whether in this PR or in a future one).

Reply:
So based on my initial assumptions:

> Does this mean that external (non mapping-team) reviewers count?
> Does this mean that all mapping team members (not just those in the reviewers subteam) are counted?

I have not made a distinction between Parchment mapping reviewers and general members of Parchment - whichever team makes more sense to be used (probably mapping reviewers, which was not fully clarified at the time this document was written), makes sense here.

> What happens if we reasonably can't get the votes of a majority of the reviewers due to lack of activity on their part?

The the status quo (PR not merged) is maintained.

Reply:
For the first point, we should clarify that "Parchment team reviewers" means those part of the @ParchmentMC/mapping-reviewers subteam, which is expected to be those team members who wish to actively review PRs (and so hold the privilege of being able to merge those PRs in), while "all reviewers" means those part of the broader @ParchmentMC/mappings team, which holds all team members who manage the mappings data (but also includes those who aren't in the mapping-reviewers team, meaning those who are not able to review PRs due to time or personal constraints).

For the second point, since the mapping-reviewers subteam should only have those willing to be active in managing PRs and reviews, it should be fine. While the first vote will only be voted upon by the mapping-reviewers, those not voting can still express their own opinion, and those voting should take these statements into account in their votes.